dependencies/pom.xml

```diff
@@ -24,6 +24,17 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
No, this needs to be a standard dependency.

Reply:
I see it cannot because com.google.auto.value.AutoValue is imported by the abstract class and com.google.auto.value.AutoValue is part of auto-value-annotations